build.sbt

```diff
@@ -52,7 +52,7 @@ lazy val scalaAwsUtils = (project in file("."))
       description := "Utilities for interacting with the AWS SDKs from Scala"
     )
   } ++ commonSettings ++ bintraySettings: _*)
-  .dependsOn(testkit)
+  .dependsOn(testkit % Test)
```

Comment:
This is the meaningful change—the rest of the commit is just cleanup. Without this scope narrowing, the main artifact includes all the test libraries (e.g. specs2 and its dependencies) which bloats things considerably.